content/200-orm/050-overview/100-introduction/250-should-you-use-prisma.mdx

```diff
@@ -103,7 +103,7 @@ _Alternatives_: [Hasura](https://hasura.io/), [Postgraphile](https://www.graphil
 
 ### ... you want to use raw, type-safe SQL for querying your database
 
-While Prisma ORM does allow you to [send plain SQL queries](/orm/prisma-client/queries/raw-database-access/raw-queries) to your database, it might not be the best fit if you prefer to work with a SQL-based abstraction that you want to be type-safe. Prisma ORM's main benefit is to provide an abstraction layer that makes you more productive compared to writing SQL.
+While Prisma ORM does allow you to [send plain SQL queries](/orm/prisma-client/using-raw-sql/raw-queries) to your database, it might not be the best fit if you prefer to work with a SQL-based abstraction that you want to be type-safe. Prisma ORM's main benefit is to provide an abstraction layer that makes you more productive compared to writing SQL.
```

Comment:
```suggestion
While Prisma ORM does allow you to [send plain SQL queries](/orm/prisma-client/using-raw-sql/raw-queries) to your database, it might not be the best fit if you prefer to work with a SQL-based abstraction that is type-safe. Prisma ORM's main benefit is to provide an abstraction layer that makes you more productive compared to writing SQL directly.
```